pkgs/development/libraries/glibc/common.nix

```diff
@@ -186,7 +187,7 @@ stdenv.mkDerivation ({
     configureScript="`pwd`/../$sourceRoot/configure"
 
     ${lib.optionalString (stdenv.cc.libc != null)
-      ''makeFlags="$makeFlags BUILD_LDFLAGS=-Wl,-rpath,${stdenv.cc.libc}/lib"''
+      ''makeFlags+=("BUILD_LDFLAGS=-Wl,-rpath,${stdenv.cc.libc}/lib")''
```

Comment:
`makeFlags` -> `makeFlagsArray` ?

Reply:
makeFlags is an array now too

makeFlagsArray will be deprecated as it is of no additional use with structuredAttrs